src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -393,14 +416,20 @@ public void elementTextShouldBe(String locator, String text, String...params) {
             "In contrast to `Element Should Not Contain`, this keyword does not try a substring match but an exact match on the element identified by locator.\r\n" + 
             "\r\n" + 
             "Key attributes for arbitrary elements are id and name. See `Introduction` for details about locators.")
-	@ArgumentNames({ "locator", "text", "message=NONE" })
+	@ArgumentNames({ "locator", "text", "message=NONE", "ignore_case=False" })
 	public void elementTextShouldNotBe(String locator, String text, String...params) {
         String message = robot.getParamsValue(params, 0, "");
 		List<WebElement> elements = elementFind(locator, true, true);
+		Boolean ignoreCase = robot.getParamsValue(params, 1, Boolean.FALSE);
 		String actual = elements.get(0).getText();
+		String textToCompare = text;
+		if (ignoreCase) {
+			actual = actual.toLowerCase();
+			textToCompare = textToCompare.toLowerCase();
+		}
 
-		if (text.equals(actual)) {
-			if (message == null || message.equals("")) {
+		if (textToCompare.equals(actual)) {
+			if (StringUtils.isNotBlank(message)) {
```

Comment:
Technically, this looks like a backward incompatible change because isNotBlank isn't the same as isNotEmpty.

Reply:
I'm fine with using isNotEmpty instead.